cmd/mv.go

```diff
@@ -14,19 +14,37 @@
 
 package cmd
 
-import "github.com/spf13/cobra"
+import (
+	"fmt"
 
-func mv(cmd *cobra.Command, args []string) (err error) {
-	arg, err := makeRelocationArg(args[0], args[1])
-	if err != nil {
-		return
-	}
+	"github.com/spf13/cobra"
+)
+
+func mv(cmd *cobra.Command, args []string) error {
+	var destination string
+	var argsToMove []string
 
-	if _, err = dbx.Move(arg); err != nil {
-		return
+	if len(args) > 2 {
+		destination = args[len(args)-1]
+		argsToMove = args[0 : len(args)-1]
+	} else if len(args) == 2 {
+		destination = args[1]
+		argsToMove = append(argsToMove, args[0])
+	} else {
+		return fmt.Errorf("mv command requires a object and a destination")
 	}
 
-	return
+	for _, argument := range argsToMove {
+		arg, err := makeRelocationArg(argument, destination+"/"+argument)
+		if err != nil {
+			return err
```

Comment:
Should we add an option that allows skipping errors? Or, if we bail on errors, should we provide a better indication of progress (e.g. while files/folders were successfully moved, which failed etc)?